examples/ota-requestor-app/efr32/src/AppTask.cpp

```diff
@@ -511,3 +526,23 @@ void AppTask::DispatchEvent(AppEvent * aEvent)
 }
 
 void AppTask::UpdateClusterState(void) {}
+
+void AppTask::InitOTARequestor()
+{
+
+    // Initialize and interconnect the Requestor and Image Processor objects -- START
+    SetRequestorInstance(&gRequestorCore);
+
+    gRequestorCore.Init(&(chip::Server::GetInstance()), &gRequestorUser, &gDownloader);
```

Comment:
FYI, after https://github.com/project-chip/connectedhomeip/pull/13849 merges, it shouldn't matter when this is called (could be before `Server::Init`) but probably still a good idea to have a separate Init function for OTA core.